src/main/java/uk/gov/hmcts/probate/model/Constants.java

```diff
@@ -97,6 +98,10 @@ public final class Constants {
     // and possibly use more constants such as VERSION_TC_INTRODUCED, and use
     // >, >=, <=, < comparisions etc, converting to numeric
     public static final String LATEST_SCHEMA_VERSION = "2.0.0";
+    public static final Long MOVED_INTO_DORMANT_TIME_ELAPSED = 5L;
```

Comment:
I thought we were trying to add these new statics into app.properties?
Also, the names are not quite right for me

Reply:
Added in the application.yaml and changed names